src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -792,6 +794,25 @@
         </xsd:sequence>
     </xsd:complexType>
 
+
+    <xsd:complexType name="mailer_dkim_signer" mixed="true">
+        <xsd:sequence>
+            <xsd:element name="options" type="metadata" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:attribute name="key" type="xsd:string" use="required" />
+        <xsd:attribute name="domain" type="xsd:string" use="required" />
+        <xsd:attribute name="select" type="xsd:string" use="required" />
+        <xsd:attribute name="passphrase" type="xsd:string" />
+    </xsd:complexType>
+
+    <xsd:complexType name="mailer_smime_signer" mixed="true">
+        <xsd:attribute name="key" type="xsd:string" use="required" />
+        <xsd:attribute name="certificate" type="xsd:string" use="required" />
```

Comment:
`use=required` should be removed. We don't require *each* XML config file to provide the attribute. We expect the final merged config to have them.